### PR TITLE
Implemented error handling by default for Swift

### PR DESCRIPTION
**Pull request by jkaster**
- `sdk.ok()` now throws if an error is encountered
- created a `listGetter` test harness like the one for Kotlin
- updated `methodsTests.swift` to use `listGetter`

### CODE REVIEW

File: swift/looker/Tests/lookerTests/methodsTests.swift [hunk 1]
```diff
@@ -85,78 +85,94 @@ class methodsTests: XCTestCase {
         } else {
             XCTAssertTrue(false, "Couldn't cast data from jsonData")
         }
+                
+        } catch {
+            print(error)
```
Comment (rlm1023) [inline swift/looker/Tests/lookerTests/methodsTests.swift]: Might even throw it to fail the test? Not sure what would happen here if it failed.

File: swift/looker/Tests/lookerTests/baseTransportTests.swift [hunk 2]
```diff
@@ -15,6 +15,7 @@ fileprivate let repoPath : String = testRootPath + "/../../"
 fileprivate let localIni : String = ProcessInfo.processInfo.environment["LOOKERSDK_INI"] ?? (repoPath + "looker.ini")
 
 
+@available(OSX 10.15, *)
```
Comment (rlm1023) [inline swift/looker/Tests/lookerTests/baseTransportTests.swift]: Interesting that these appeared/changed?

File: swift/looker/Tests/lookerTests/methodsTests.swift (hunk 1 above)
Comment (jkaster) [reply swift/looker/Tests/lookerTests/methodsTests.swift]: I think I'll leave it with the current pattern for now. Still learning Swift practices.

File: swift/looker/Tests/lookerTests/baseTransportTests.swift (hunk 2 above)
Comment (jkaster) [reply swift/looker/Tests/lookerTests/baseTransportTests.swift]: It's because I pushed more initialization into `TestConfig()` that requires 10.15

**Pull request merged**